Makefile.am

```diff
@@ -161,13 +161,13 @@ src_tpm2_send_command_SOURCES = src/tpm2_send_command.c src/main.c
 src_tpm2_startup_SOURCES = src/tpm2_startup.c src/main.c
 
 if UNIT
-test_tpm2_rc_decode_unit_CFLAGS  = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
-test_tpm2_rc_decode_unit_LDADD   = $(CMOCKA_LIBS)
-test_tpm2_rc_decode_unit_SOURCES = test/tpm2-rc-decode_unit.c
+test_unit_tpm2_rc_decode_unit_CFLAGS  = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
+test_unit_tpm2_rc_decode_unit_LDADD   = $(CMOCKA_LIBS)
+test_unit_tpm2_rc_decode_unit_SOURCES = test/unit/tpm2-rc-decode_unit.c
 
-test_tpm2_rc_entry_unit_CFLAGS   = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
-test_tpm2_rc_entry_unit_LDADD    = $(CMOCKA_LIBS)
-test_tpm2_rc_entry_unit_SOURCES  = src/rc-decode.c test/tpm2-rc-entry_unit.c
+test_unit_tpm2_rc_entry_unit_CFLAGS   = $(AM_CFLAGS) $(CMOCKA_CFLAGS)
+test_unit_tpm2_rc_entry_unit_LDADD    = $(CMOCKA_LIBS)
+test_unit_tpm2_rc_entry_unit_SOURCES  = src/rc-decode.c test/unit/tpm2-rc-entry_unit.c
```

Comment:
Some efficiency here would be nice. Unfortunately if you start doing interesting things with cmocka (like using the `-Wl,--wrap`for mocking) then you'll need to provide LDFLAGS for the unit tests. Since we're not doing anything like this (yet?) we may be able to get away with what you describe but we may be placing constraints on ourselves that make future work more difficult.